src/OpenCL.cpp

```diff
@@ -260,6 +258,150 @@ __kernel void out_transform_fused_bn(__global float *M,
         }
     }
 }
+
+__kernel void out_transform_fused_bn_in(
+                                     __global float *M,
+                                     __global net_t *Y,
+                                     __global net_t *V,
+                                     const int K,
+                                     const int Kpad, const int Ppad, const int Cpad,
+                                     __global const net_t * residual,
+                                     __constant const net_t * means,
+                                     __constant const net_t * stddivs,
+                                     __local float *ybuf) {
+    const int W = 19;
+    const int H = 19;
+    const int T = W*H;
+    const int WTILES = (W + 1) / 2;
+    const int P = WTILES * WTILES;
+    const int KPpad = Kpad * Ppad;
+
+    const int k = get_global_id(0);
+    const int kg = get_local_id(0);
+    const int block = get_global_id(1);
+    const int chT = k*(T);
+
+    const int block_x = block % WTILES;
+    const int block_y = block / WTILES;
+
+    const int yin = 2 * block_y - 1;
+    const int xin = 2 * block_x - 1;
+
+
+    const int x = 2*block_x;
+    const int y = 2*block_y;
+    int a_ind = (y)*W + (x);
+
+    const int kHW = k * W * H;
```

Comment:
Is this the same as chT? or do the brackets around T do something I don't know about?